### ISSUE TITLE
Feature request: Home folder

### ISSUE THREAD

**Issue by mattemc** [conceptualspace/yet-another-speed-dial#45]
I have just migrated from another speed dial add-on. There during many years I have accumulated numerous links and folders. The migration was smooth, but I noticed that the links displayed in the home folder are held in the Speed Dial folder rather than in a dedicated subfolder named "Home". The large number of links that I saved in the Home folder together with the large number of sub-folders, made my Speed Dial folder quite messy.
I suppose that my experience can represent a plausible scenario for many other users as their collection of links grows. Wouldn't be better to save all the links in sub-folders, included the links in the Home folder? 
**- Might shipping the add-on with a default ~/Speed Dial/Home sub-folder in the bookmarks be a good idea?** 
Thank you very much for the good work ;)

**Comment by conceptualspace** [comment]
yes, i agree in general, certainly for those who use folders. my only hesitation is for those who don't use folders -- they end up with a deeper hiercharchy in the bookmarks manager for seemingly no reason (~/Other Bookmarks/Speed Dial/Home) and might be confused what the home subfolder is for. i suppose the question is which is the bigger inconvenience, and to how many users?

secondly, if we decide to use a home folder, how to we manage that during upgrade? silently move the bookmarks from the root speed dial folder into the home subfolder? maybe a prompt and confirmation button?

**Comment by SerafDosSantos** [comment]
> silently move the bookmarks from the root speed dial folder into the home subfolder? maybe a prompt and confirmation button?

@conceptualspace sounds good, I wouldn't mind

**Comment by mattemc** [comment]
> silently move the bookmarks from the root speed dial folder into the home subfolder? maybe a prompt and confirmation
@conceptualspace Very neat solution, sound good to me too...

I was thinking to map the Home to both folders: ~/Speed Dial/Home  and  ~/Speed Dial  to accommodate both needs. This solution may allow the tidy users to arrange their bookmarks into deeper hierarchies, while the casual ones don't have to deal with the extra sub-folder. At the same time, probably that ~/Speed Dial/Home is going to confuse someone at some point...

**Comment by conceptualspace** [comment]
yeah i like that. maybe we can just default to the root until a folder is created, if subfolders exist we map everything to home: that matches the user experience on the front end. and as a bonus we can enable renaming the 'home' folder (right now it's just a static label)

just need to find some time but i'll target to have this working within the next few releases

**Comment by rocvfx** [comment]
Now there are more and more tags in the collection, it is very difficult to sort them, it would be great if there is a sub-folder function, thanks!
